constrictor_test.go

```diff
@@ -45,89 +45,67 @@ func TestConstrictorModifyInitialBoardState(t *testing.T) {
 	}
 }
 
-func TestConstrictorCreateNextBoardState(t *testing.T) {
-	tests := []struct {
-		prevState     *BoardState
-		moves         []SnakeMove
-		expectedState *BoardState
-	}{
-		{
-			&BoardState{
-				Width:  3,
-				Height: 3,
-				Snakes: []Snake{
-					{
-						ID:     "one",
-						Body:   []Point{{0, 0}, {0, 0}, {0, 0}},
-						Health: 100,
-					},
-					{
-						ID:     "two",
-						Body:   []Point{{2, 2}, {2, 2}, {2, 2}},
-						Health: 100,
-					},
-				},
-				Food: []Point{},
+// Test that two equal snakes collide and both get eliminated
+// also checks:
+//	- food removed
+//  - health back to max
+var constrictorMoveAndCollideMAD = gameTestCase{
```

Comment:
Probably just missing something obvious here, but what does MAD stand for here and in the other test cases?

Reply:
MAD = Mutually Assured Destruction. Should I change that or is it okay?